cmd/edgectl/aes_install.go

```diff
@@ -350,3 +404,9 @@ spec:
   acmeProvider:
     email: %s
 `
+
+// FIXME: Mention that this will be shared with Let's Encrypt?
```

Comment:
@richarddli

Reply:
If it will be shared, then we should mention it.